quinn-proto/src/connection.rs

```diff
@@ -1230,7 +1231,7 @@ impl Connection {
                         }
                     }*/
                     Header::Long { ty, .. } => {
-                        debug!(ctx.log, "unexpected packet type"; "type" => format!("{:02X}", ty));
+                        debug!(ctx.log, "unexpected packet type"; "type" => format!("{:?}", ty));
                         Err(TransportError::PROTOCOL_VIOLATION.into())
                     }
```

Comment:
Makes sense.

Reply:
On second thought, I think it's correct to just drop 0-RTT packets when we don't implement them, so we should probably just log a message at debug level and leave the state as-is rather than panicing. That way we can be more robust in our initial draft-15 support.

Reply:
Done.